common.yml

```diff
@@ -5,15 +5,15 @@ services:
       context: ./src/django
       dockerfile: Dockerfile
     environment:
-      - PGHOST=database.service.pfb.internal
-      - PGDATABASE=pfb
-      - PGPASSWORD=pfb
-      - PGPORT=5432
-      - PGUSER=pfb
       - DJANGO_ENV=development
-      - DJANGO_SECRET_KEY=secret
       - DJANGO_LOG_LEVEL=INFO
       - AWS_PROFILE=pfb
+      - PFB_DB_HOST=database.service.pfb.internal
```

Comment:
This actually brings this file in line with what was already in the task definition files.  I think those were working before because of fallbacks/defaults in settings.py.